scripts/biomedical/ENCODE/parser.py

```diff
@@ -0,0 +1,929 @@
+"""Parses files scraped from the Encode project into either an intermediate.
```

Comment:
I will point to an earlier version of this without g3 hooks.

This also needs license.